ui/app/helpers/utils/util.js

```diff
@@ -465,18 +465,31 @@ export function constructTxParams({
  * or throws an error in case of failure.
  */
 export async function jsonRpcRequest(rpcUrl, rpcMethod, rpcParams = []) {
+  let fetchUrl = rpcUrl
+  const headers = {
+    'Content-Type': 'application/json',
+  }
+  // Convert basic auth URL component to Authorization header
+  const { origin, pathname, username, password } = new URL(rpcUrl)
+  // URLs containing username and password needs special processing
+  if (username && password) {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Is this line still needed? :thinking: I don't see any unused vars

Reply:
Indeed, just removed it in the latest commit